### PR TITLE
Election 2015 badging

### PR DESCRIPTION
**Pull request by phamann**
This is a temporary (for the next couple of months) hack, to introduce our election logo on all content tagged with the election tag as the primary keyword. This is so we have consistency across all mediums print, web app for our coverage. 

It's been a contentious debate, and we have settled as a feature switch being the most appropriate solution.
### Screenshots

google chrome

google chrome

littlesnapper

google chrome

/cc @JustinPinner @gklopper @stephanfowler

### CODE REVIEW

**Comment by rich-nguyen** [comment]
:+1:

File: common/app/views/fragments/meta/metaInline.scala.html [hunk 1]
```diff
@@ -1,22 +1,37 @@
 @(content: model.Content)(implicit request: RequestHeader)
 
 @import common.{LinkTo, Localisation}
+@import conf.Switches.ElectionLiveBadgeSwitch
 
-<div class="content__labels">
-    <div class="content__section-label">
+@defining(
+    ElectionLiveBadgeSwitch.isSwitchedOn &&
+    (content.tags.indexWhere(_.id == "politics/general-election-2015") == 0 ||
```
Comment (gklopper) [inline common/app/views/fragments/meta/metaInline.scala.html]: `content.tags.headOption.exists(_.id == "politics/general-election-2015")` maybe

**Comment by gklopper** [comment]
:+1:  -  but I think judicious use of the `ElectionLiveBadgeSwitch ` would make it easier to delete

**Comment by gklopper** [comment]
Also the red on red of the live blog and badge looks terrible...

**Pull request merged**